src/components/AnnouncementBanner.tsx

```diff
@@ -25,6 +27,28 @@ const CenteredBannerText = styled.div`
 export const AnnouncementBanner = () => (
   <SBanner
     type={BannerType.ANNOUNCEMENT}
-    value={<CenteredBannerText>{translate('LAUNCH_ANNOUNCEMENT')}</CenteredBannerText>}
+    value={
+      <CenteredBannerText>
+        <Trans
+          id="LAUNCH_ANNOUNCEMENT"
+          variables={{
+            $link: () => (
+              <LinkApp
+                href={MYCRYPTO_PROD_LINK}
+                isExternal={true}
+                variant="underlineLink"
+                $underline={true}
+              >
+                {
+                  // Remove protocol from uri
```

Comment:
Merely for display reasons so it can read:

<img width="748" alt="Screenshot 2021-02-10 at 15 46 23" src="https://user-images.githubusercontent.com/2429708/107525435-23dff880-6bb7-11eb-89a5-e641ea108293.png">

vs `https://app.mycrypto.com`